interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -0,0 +1,13 @@
+<div class="well">
+  <h3>
+    Machine Learning Model
+  </h3>
```

Comment:
Exactly. `gene-network.tpl.html` is an example of the *layout* in the separation of concerns I'm trying to establish. Ideally, everything in that file should focus exclusively on assembling *modules* (or *components*, to use the specific Angular example) and where they fall in the flow of the page, how they resize with the window, etc.